code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -21,7 +21,7 @@
 	/area/turret_protected/ai, /area/storage/emergency, /area/storage/emergency2, /area/crew_quarters/sleep, /area/security/brig, /area/shuttle)
 	target_trait = STATION_LEVEL
 
-	immunity_type = "rad"
+	immunity_type = RAD
```

Comment:
This one needs to stay as just "rad" I think:
![image](https://user-images.githubusercontent.com/57483089/130305610-a52851c5-26fd-4a2c-86e3-b3986a8edbec.png)
```suggestion
	immunity_type = "rad"
```